### PR TITLE
Changing to use bcrypt-ed tokens for auth

### PR DESCRIPTION
**Pull request by FishNecks**

### CODE REVIEW

File: api/middleware_test.go [hunk 1]
```diff
@@ -5,10 +5,14 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	
+	log "github.com/sirupsen/logrus"
```
Comment (fishnix) [inline api/middleware_test.go:9]: looks like this library is unused in the tests so its causing the test to fail

Comment (fishnix) [reply api/middleware_test.go:9]: (it was probably used in the benchmarks of the other codebase)

Comment (FishNecks) [reply api/middleware_test.go]: Yep, fixed.

**Comment by fishnix** [comment]
👍

**Pull request merged**